schemas/context/direct-marketing.schema.json

```diff
@@ -24,6 +24,11 @@
           "description":
             "Click(s) actions on a message."
         },
+        "xdm:offerClicks": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description":
+            "Click on an offer in a message."
```

Comment:
I would remove "in a message". Keep it more open.

Reply:
More and more people tap instead of click ;)

Reply:
renamed the measure property to offerOpens to reflect that an offer doesn't need a click to be counted as "interacted"